docs/platform/porting/recovery.md

```diff
@@ -36,492 +36,10 @@ the `/system` directory, and other filesystems (such as `/opt` and
-       actions = ({
-           label = "Reboot system now";
-           screen_switch_to = "reboot";
-       },{
-           label = "Safe mode";
-           screen_switch_to = "safe";
-       },{
-           label = "Phone reinitialisation";
-           screen_switch_to = "factory";
-       },{
-           label = "Minimal phone reinitialisation";
-           screen_switch_to = "factory-minimal";
-       });
-   };
-   ```
-
-## Look-and-feel Options
-
-The following example shows the remaining parts of the configuration file, containing options that control the look-and-feel of the system recovery menus:
-
-```
-colors = {
-    background = "#000000ff";
-    title = "#1bc7ccff";
-    ruler = "#1bc7ccff";
-    white = "#ffffffff";
-};
-
-ruler_styles = {
-    common = {
-        c_ruler = "ruler";
-    };
-};
-
-menu_styles = {
-    common = {
-        item_height = 80;
-        item_spacing = 8;
-        text_pos_x = 15;
-        c_bg_selected = "title";
-        c_bg_unselected = "background";
-        c_text_selected = "white";
-        c_text_unselected = "white";
-    };
-};
-
-screen_styles = {
-    common = {
-        c_background = "background";
-    };
-};
-
-description_styles = {
-    common = {
-        c_title = "title";
-        c_text = "white";
-    };
-};
-
-rulers = {
-    main = (
-    {
-        pos_x = 0;
-        pos_y = 80;
-        height = 2;
-        style = "common";
-    }, {
-        pos_x = 0;
-        pos_y = 420;
-        height = 2;
-        style = "common";
-    });
-
-    confirm = (
-    {
-        pos_x = 0;
-        pos_y = 80;
-        height = 2;
-        style = "common";
-    });
-};
-
-images = {
-    background_default = {
-        fname = "/usr/share/system-recovery/res/images/warning.png";
-        c_bg = "background"; // reference to colors
-        align_hor = "center";
-        align_ver = "bottom";
-        offset_x = 0;
-        offset_y = 0;
-        img_type = "alpha"; // alt: "no-alpha";
-    };
-    menu_title = {
-        fname = "/usr/share/system-recovery/res/images/menu-title.png";
-        c_bg = "title";
-        align_hor = "center";
-        align_ver = "top";
-        offset_x = 0;
-        offset_y = 20;
-        img_type = "no-alpha"; // alt: "no-alpha";
-    };
-};
-
-animations = {
-    working = {
-        fname = "/usr/share/system-recovery/res/images/tizen-anim.png";
-        c_bg = "background";
-        align_hor = "center";
-        align_ver = "middle";
-        offset_x = 0;
-        offset_y = 0;
-        frames_num = 0;
-        current_frame = 0;
-        img_type = "no-alpha"; // alt: "no-alpha";
-    };
-};
-```
+4. The `system-recovery` program search `tizen-recovery.img` from removable storage.
+`tizen-recovery.img` has compressed partition image and config file.
+`tizen-recovery.img` is found from removable storage, `system-recovery` program decompress `tizen-recovery.img`
+and dump partition image to appropicate block with config file.
```

Comment:
Suggest to rephrase, please check if the below statement is technically accurate.

The `system-recovery` program searches the removable storage for `tizen-recovery.img` which has the  compressed partition image and config file. The `system-recovery` program decompress `tizen-recovery.img`, and dumps the partition image to the appropriate block with config file.


Have a question here.. does the config file have the block information, or the config file is copied to the block along with the partition image?

Reply:
> Suggest to rephrase, please check if the below statement is technically accurate.

Thank you for suggestion. I looks good, i will fix it 👍 

> Have a question here.. does the config file have the block information, or the config file is copied to the block along with the partition image?

Config file have block information. `system-recovery` program just use it for find proper block.

Reply:
@sh95jeon  
Thank you for clarifying regarding the config file, can the second line be modified like this:

The `system-recovery` program decompress `tizen-recovery.img`, and dumps the partition image to the appropriate block as mentioned in the config file.

Reply:
@annie-samsung Thank you for reviewing 😄 I fixed it :)